File: app/src/main/java/org/fossasia/susi/ai/rest/model/LoginResponse.java
package org.fossasia.susi.ai.rest.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by saurabh on 12/10/16.
 */

public class LoginResponse {
    @SerializedName("message")
    @Expose
    String message;
    @SerializedName("session")
    @Expose
    Session session;
    @SerializedName("valid_seconds")
    @Expose
    Long validSeconds;
    @SerializedName("access_token")
    @Expose
    String accessToken;

    public String getMessage() {
        return message;
    }

    public Session getSession() {
        return session;
    }

    public Long getValidSeconds() {
        return validSeconds;
    }

    public String getAccessToken() {
        return accessToken;
    }
}

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt
package org.fossasia.susi.ai.rest.responses.susi

import com.google.gson.annotations.SerializedName

data class LoginResponse(
    val message: String? = null,
    val session: Session? = null,
    @SerializedName("valid_seconds")
    val validSeconds: Long = 0,
    @SerializedName("access_token")
    var accessToken: String? = null
)
